nodecontrol/algodControl.go

```diff
@@ -133,7 +149,11 @@ func (nc NodeController) algodRunning() (isRunning bool) {
 }
 
 // StopAlgod reads the net file and kills the algod process
-func (nc *NodeController) StopAlgod() (alreadyStopped bool, err error) {
+func (nc *NodeController) StopAlgod() (err error) {
+	// Check for valid data director
+	if !util.IsDir(nc.algodDataDir) {
```

Comment:
This comment is for something not part of this PR, but I would like to raise it here:
Why do we say "kill node"? Shouldn't it be "stop node"? 
```
 $ goal node stop -d /tmp/a.txt 
Cannot kill node: /tmp/a.txt is not a valid directory
```
Instead:
`Cannot stop node: /tmp/a.txt is not a valid directory.`

Reply:
yes, I agree that the user-facing error message should refrain of using the "kill" terminology, since it's the signal name and not the operation.

Reply:
@algonautshant - could you open another PR for the kill vs stop ?
it's in goal,
```
errorKill                         = "Cannot kill node: %s"
```
and would affect some of the changes made here ( but maybe more )